pkg/occlient/occlient_test.go

```diff
@@ -507,6 +507,14 @@ func Test_parseImageName(t *testing.T) {
 			want3:   "",
 			wantErr: false,
 		},
+		{
+			arg:     "",
```

Comment:
The tests LGTM, but just one minor comment: The output looks a bit odd without a name for test cases:

```
=== RUN   Test_parseImageName/image_name:_nodejs
=== RUN   Test_parseImageName/image_name:_
=== RUN   Test_parseImageName/image_name:_:
```  

It might be good to have a name for each test case, something like "Case: empty image name". Check the other tests, you will get what I am saying.

Reply:
Test cases have names it is called "image name: " + `whatever image test name is` 😉 
I just felt that adding name field and naming each test is just cumbersome and not needed as it can be generated (to lower a barrier to add new test cases)

Reply:
Done